qiskit_experiments/library/characterization/analysis/readout_angle_analysis.py

```diff
@@ -14,31 +14,92 @@
 """
 
 import numpy as np
+from typing import List, Optional
 
-from qiskit_experiments.framework import BaseAnalysis, AnalysisResultData
-
+from qiskit_experiments.framework import BaseAnalysis, AnalysisResultData, Options
+from qiskit_experiments.framework.matplotlib import get_non_gui_ax
 
 class ReadoutAngleAnalysis(BaseAnalysis):
     """
     A class to analyze readout angle experiments
     """
+    
+    @classmethod
+    def _default_options(cls) -> Options:
+        """Return default analysis options.
+
+        Analysis Options:
+            plot (bool): Set ``True`` to create figure for fit result.
+            ax(AxesSubplot): Optional. A matplotlib axis object to draw.
+        """
+        options = super()._default_options()
+        options.plot = True
+        options.ax = None
+        return options
 
-    def _run_analysis(self, experiment_data):
+    def _run_analysis(self, experiment_data, **kwargs):
```

Comment:
I think nowadays we don't have kwargs here anymore

Reply:
Right, I removed the kwargs after lint complained about it.